agent/command/git.go

```diff
@@ -791,9 +784,7 @@ func (c *gitFetchProject) fetchModuleSource(ctx context.Context,
 
 		errOutput := stdErr.String()
 		if errOutput != "" {
-			if opts.token != "" {
-				errOutput = strings.Replace(errOutput, opts.token, "[redacted github token]", -1)
-			}
+			errOutput = strings.ReplaceAll(errOutput, "\n", fmt.Sprintf("\n%s: ", module.Name))
```

Comment:
I added this line to append every line with the module name for easier debugging